tests/kuttl/tests/basic-deployment/04-rotate-service-certs.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |
+      echo "Waiting for OpenStack control plane to be ready..."
+      oc wait openstackcontrolplane -n $NAMESPACE --for=condition=Ready --timeout=400s -l core.openstack.org/openstackcontrolplane
+
+  - script: |
+      echo "Get fingerprints of all service certs"
+      oc exec -i openstackclient -n $NAMESPACE -- bash -s < ../../common/osp_endpoint_finger_prints.sh > /tmp/endpoint_fingerprints_before
```

Comment:
Maybe we should use the absolute path instead of relative? I know elsewhere we use relative path successfully, but since this is falling we can set the path something like I did in here - https://github.com/openstack-k8s-operators/openstack-operator/commit/f7edf60d3d1cdcd8f465a9983e79add29a49f5d8

Reply:
calling the script here in the TestStep is not the issue. the issue is in the TestAssert, see the comment there